modules/benchmark/R/load.netcdf.R

```diff
@@ -0,0 +1,29 @@
+load.netcdf <- function(path,vars,units,newunits){
```

Comment:
I like the use of oldunits and newunits.

Is there overlap between this function and [`get.ncvector`](https://github.com/PecanProject/pecan/blob/master/modules/data.atmosphere/R/temporal.downscaling.R#L215) (which I wrote specifically for met data ...), e.g.

```r
lapply(variables, function(x) get.ncvector(x, lati = lati, loni = loni, 
                                                     run.dates = run.dates, met.nc = nc))
```

I'm not actually sure how these would be combined, but wanted to point out that it would be great to have a canonical, robust method of extracting and transforming data from netcdf files ...